### PR TITLE
Legacy versions - Closes #264

### LINKED ISSUE #264: Legacy versions of the SDK
### Description

Make sure the legacy versions of the SDK are supported by Lisk Service.

### Motivation

- Making sure the Lisk Core works properly with the SDKv2, SDKv3 and SDKv4 is required to finish the alpha phase.

### Acceptance Criteria

- Legacy versions work properly
- Testnet tests are passing

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #264 

### How was it solved?
- [x] SDKv2 works properly + Integration tests are passing
- [x] SDKv3 works properly + Integration tests are passing
- [x] SDKv4 works properly + Integration tests are passing
- [x] Set default values of `limit` and `offset` in gateway

### How was it tested?
Local

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM!

File: tests/integration/api_v1/http/transactions.test.js [hunk 1]
```diff
@@ -269,12 +267,15 @@ describe('Transactions API', () => {
 
 	describe('Retrieve transaction list within timestamps', () => {
 		it('transactions within set timestamps are returned', async () => {
-			const from = moment(refTransaction.timestamp).subtract(1, 'day').unix() * (10 ** 3);
-			const toTimestamp = refTransaction.timestamp;
+			const transactions = (await api.get(`${endpoint}`)).data;
```
Comment (MichalTuleja) [inline tests/integration/api_v1/http/transactions.test.js:270]: Is there any reason to use `${endpoint}` instead of just endpoint?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: tests/integration/api_v1/http/transactions.test.js (hunk 1 above)
Comment (nagdahimanshu) [reply tests/integration/api_v1/http/transactions.test.js]: Updated it to `endpoint`. Thanks!

**Review by MichalTuleja — APPROVED** [review]
Looks good, thanks! 👍

**Pull request merged**